### ISSUE TITLE
Shutting Windows down while Halite is running causes all settings (including hash database) to be reset

### ISSUE THREAD

**Issue by apexlee** [Eoinocal/Halite#24]
Hi,

So I decided to give Halite a whirl as it looks very minimalistic and have come across this issue.

It appears there is no saving mechanism to the settings file while Halite is open, only on close.

Things I've changed:
- Enabled/Disabled some columns
- Resized columns
- Hashed/checked torrents that have been imported

When I shut down or restart Windows while Halite is running, it wipes this and I have start again. However, I did notice the settings values are saved.

Expected Result - Halite should save these changes not only after a safe close, but during the session.

PS - Keep up the great work!

**Comment by Eoinocal** [comment]
Yeah, this really annoys me too :-) So a fix is high on my priority list. That said tracking it down is tricky, having to shutdown your machine to test every change gets tedious very quickly.

**Comment by Zero3K** [comment]
I hope this gets fixed soon.
